static/src/stylesheets/module/external/_witness.scss

```diff
@@ -69,54 +69,38 @@
     margin-bottom: 0;
 }
 
-.witness-cta-wrapper {
-    position: relative;
+.witness-cta {
     overflow: hidden;
-    height: 47px;
     margin-bottom: ($gs-baseline/3)*4;
 }
-.witness-cta__body {
-    @include fs-textSans(3);
-    float: left;
-    line-height: 15px;
-    padding: $gs-baseline/2 $gs-gutter/2 0;
-    margin-left: 50px;
 
-    .witness-cta__logo {
-        display: block;
-    }
+.button--witness {
```

Comment:
Yes. And the thing is that we shouldn't anyway as this button is a very specific object for only .com project and button component should be for a button abstraction which can be used across the whole organisation.

That's one of the reasons why I would recommend not using '.button--witness' but '.witness__button' thus making it **element of the witness object instead** of **modification of the button object**.

I am happy to explain it more in person if you like.